features/api_services_list.feature

```diff
@@ -0,0 +1,395 @@
+              "entitled": null,
+              "name": "fips"
+            },
+            {
+              "available": false,
+              "description": "Preview of FIPS crypto packages undergoing certification with NIST",
+              "entitled": null,
+              "name": "fips-preview"
+            },
+            {
+              "available": false,
+              "description": "FIPS compliant crypto packages with stable security updates",
+              "entitled": null,
+              "name": "fips-updates"
+            },
+            {
+              "available": true,
+              "description": "Management and administration tool for Ubuntu",
+              "entitled": null,
+              "name": "landscape"
+            },
+            {
+              "available": true,
+              "description": "Canonical Livepatch service",
+              "entitled": null,
+              "name": "livepatch"
+            },
+            {
+              "available": true,
+              "description": "Ubuntu kernel with PREEMPT_RT patches integrated",
+              "entitled": null,
+              "name": "realtime-kernel"
+            },
+            {
+              "available": false,
+              "description": "Security Updates for the Robot Operating System",
+              "entitled": null,
+              "name": "ros"
+            },
+            {
+              "available": false,
+              "description": "All Updates for the Robot Operating System",
+              "entitled": null,
+              "name": "ros-updates"
+            }
+          ]
+        },
+        "meta": {
+          "environment_vars": []
+        },
+        "type": "ServicesList"
+      }
+      """
+    When I attach `contract_token` with sudo
+    And I run `pro api u.pro.services.list.v1` as non-root
+    Then API data field output matches regexp:
+      """
+      {
+        "attributes": {
+          "services": [
+            {
+              "available": true,
+              "description": "Scalable Android in the cloud",
+              "entitled": true,
+              "name": "anbox-cloud"
+            },
+            {
+              "available": false,
+              "description": "Common Criteria EAL2 Provisioning Packages",
+              "entitled": true,
+              "name": "cc-eal"
+            },
+            {
+              "available": true,
+              "description": "Expanded Security Maintenance for Applications",
+              "entitled": true,
+              "name": "esm-apps"
+            },
+            {
+              "available": true,
+              "description": "Expanded Security Maintenance for Infrastructure",
+              "entitled": true,
+              "name": "esm-infra"
+            },
+            {
+              "available": false,
+              "description": "NIST-certified FIPS crypto packages",
+              "entitled": true,
+              "name": "fips"
```

Comment:
Service availability may vary between releases.
I would say the specific values here could use columns in the `Examples` table below, where you set them.
This test fails on `focal`, for instance, where FIPS is available.